romancal/assign_wcs/tests/test_wcs.py

```diff
@@ -1,16 +1,23 @@
 import numpy as np
 import pytest
+from astropy.modeling import models
 from gwcs.wcstools import grid_from_bounding_box
 from numpy.testing import assert_allclose
 from roman_datamodels import datamodels as rdm
-from roman_datamodels import maker_utils
+from roman_datamodels import stnode as st
 
 from romancal.assign_wcs.assign_wcs_step import AssignWcsStep, load_wcs
 from romancal.assign_wcs.utils import wcs_bbox_from_shape
 
+DATA_SHAPE = (100, 100)
+TEST_TRANSFORM = models.Shift(1) & models.Shift(2)
+
 
 def create_image():
-    l2 = maker_utils.mk_level2_image()
+    l2 = rdm.ImageModel.fake_data(shape=DATA_SHAPE)
+
+    l2.meta.cal_step = st.L2CalStep.fake_data()
+    l2.meta.cal_logs = st.CalLogs.fake_data()
```

Comment:
Many of these tests only work if the input model has `cal_step` and `cal_logs`. However these are not required by the schema.

If they are not required there are many bugs in the pipeline that have been hidden by the maker_utils automatically populating these. If they are required we can make them required in the schema and the single call to `ImageModel.fake_data` will add them.